autoload/gofmt.vim

```diff
@@ -66,9 +66,14 @@ function! gofmt#apply() abort
   endif
 
   let input = getline(1, '$') + ['']
-  let diff = split(system(cmd, input), "\n", 1)
+  let output = system(cmd, input)
+  let diff = split(output, "\n", 1)
   if v:shell_error != 0
-    return
+    if g:gofmt_display_errors
+      echohl ErrorMsg
+      echo output
```

Comment:
Oh sorry, I meant it should look like:

```vim
echohl ErrorMsg
echo output
echohl None
```

It's to reset the message highlight.